package.json

```diff
@@ -36,6 +36,7 @@
     "react-native-document-picker": "^3.5.4",
     "react-native-extended-stylesheet": "^0.12.0",
     "react-native-fs": "^2.16.6",
+    "react-native-geolocation-service": "^5.2.0",
```

Comment:
If you add a new module with a native part, please use `cd ios; pod install` to update the iOS project. Otherwise it won't build.

Reply:
Yes sure. I will do it in the next commit.

Reply:
Well, that probably demands having installed at least CocoaPods, so maybe you need MacOS - if you work with Linux oder Windows I can do it for you.

Reply:
Added it for you now.